vendor/github.com/pborman/uuid/.travis.yml

```diff
@@ -1,10 +0,0 @@
-language: go
```

Comment:
Thanks for deleting code and reducing our dependencies. I don't know which implementation is better, but having two sounds bogus.

Reply:
Thanks for reviewing this PR. Both `github.com/pborman/uuid` and `github.com/google/uuid` implements [RFC4122](https://datatracker.ietf.org/doc/html/rfc4122), but only `github.com/google/uuid` is still being updated regularly.